dependencies.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitrise-io/go-utils/v2/command"
+)
+
+// EnsureDependenciesOpts ...
+type EnsureDependenciesOpts struct {
+	GemVersions gemVersions
+	UseBundler  bool
+}
+
+// InstallDependencies ...
+func (f FastlaneRunner) InstallDependencies(config Config, opts EnsureDependenciesOpts) error {
+	// Install desired Fastlane version
+	if opts.UseBundler {
```

Comment:
Where does `opts.UseBundler` get a value?

Reply:
Moved it to `Config.GemVersions.fastlane.Found` and set it in ProcessConfig:
```
dependenciesOpts := EnsureDependenciesOpts{
  ...
  UseBundler:     config.GemVersions.fastlane.Found,
  ...
}
```